Documentation/ceph-dashboard.md

```diff
@@ -111,3 +111,62 @@ rook-ceph-mgr-dashboard-external-https  NodePort    10.101.209.6     <none>
 In this example, port `31176` will be opened to expose port `8443` from the ceph-mgr pod. Find the ip address
 of the VM. If using minikube, you can run `minikube ip` to find the ip address.
 Now you can enter the URL in your browser such as `https://192.168.99.110:31176` and the dashboard will appear.
+
+If you have a cluster with an [nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/)
```

Comment:
How about if we give more visibility to this section with a header and also add a header for the node port example above? I'm thinking it would look like this:
```
## Viewing the Dashboard External to the Cluster

Commonly...

There are...

### Node Port

The simplest way...

### Ingress Controller

If you have a cluster...
```

Reply:
Added the suggested headers.